README.md

```diff
@@ -52,6 +52,26 @@ make VERSION=0.3.0 push
 make clean
 ```
 
+On Windows, using the tools described below, use the command:
+
+"C:\Program Files\git\bin\bash" -c "make"
+"C:\Program Files\git\bin\bash" -c "make test"
+"C:\Program Files\git\bin\bash" -c "make test TESTARGS='-run TestSomething'"
+"C:\Program Files\git\bin\bash" -c "make gofmt"
+
+
+## Installed requirements
+
+You'll need:
+* docker-ce (will work with move versions and platforms)
+* gnu make
+* golang
+
+On windows the building is somewhat more difficult due to the build being bash/linux/make-oriented, but support is provided. You need:
```

Comment:
You could download the packages if you had *exactly* the same packages as choco is installing. I was trying to systematize the whole thing.  Unfortunately, there are a lot of versions of make around; git+git bash is a little more standardized.